Content.Server/Speech/EntitySystems/SlurredSystem.cs

```diff
@@ -61,46 +61,62 @@ private string Accentuate(string message, float scale)
         // This is pretty much ported from TG.
         foreach (var character in message)
         {
+            var modChar = character;
+
             if (_random.Prob(scale / 3f))
             {
-                var lower = char.ToLowerInvariant(character);
+                var lower = char.ToLowerInvariant(modChar);
                 var newString = lower switch
                 {
                     'o' => "u",
                     's' => "ch",
                     'a' => "ah",
                     'u' => "oo",
                     'c' => "k",
-                    _ => $"{character}",
+                    _ => $"{modChar}",
                 };
 
                 sb.Append(newString);
             }
 
             if (_random.Prob(scale / 20f))
             {
-                if (character == ' ')
+                if (modChar == ' ')
```

Comment:
Cleanup: this is an if-else chain checking against consts; make this a switch so it's easier to extend.